### PR TITLE
Closes #612

### LINKED ISSUE #612: add a section to the status page that lists where the .vent folder is and maybe where the watched folder is as well.

### PR DESCRIPTION
**Pull request by Jeff-Wang93**
Tell me what you think about the look. 
the `.vent` in the menu looks kinda out of place but I didnt know what else to call it so it would fit.

### CODE REVIEW

File: vent/menus/main.py [hunk 1]
```diff
@@ -350,6 +354,14 @@ def create(self):
                                  labelColor='DEFAULT', value=Cpu())
         self.gpufield = self.add(npyscreen.TitleFixedText, name='GPUs:',
                                  labelColor='DEFAULT', value=Gpu()[1])
+        self.location = self.add(npyscreen.TitleFixedText,
+                                 name='.vent:',
```
Comment (cglewis) [inline vent/menus/main.py]: Let's call this `User Data:`

File: vent/menus/main.py [hunk 2]
```diff
@@ -350,6 +354,14 @@ def create(self):
...
+                                 value=PathDirs().meta_dir,
+                                 labelColor='DEFAULT')
+        self.file_drop = self.add(npyscreen.TitleFixedText,
+                                  name='File Drop:',
+                                  value=template.section("main")[1][0][1],
```
Comment (cglewis) [inline vent/menus/main.py]: order can't be assumed...

Comment (Jeff-Wang93) [inline vent/menus/main.py]: I know that will return a tuple of `(bool, str)`. Maybe just display the `str`? 
So the vent menu would show `File Drop: [(files, /tmp/vent_files)]`
Is that fine?

Comment (cglewis) [inline vent/menus/main.py]: what i meant was you can't guarantee that `files` is the only option (or the first one) in the `main` section.  eventually there will be other options.  just having the path (value) is good, but it needs to specifically grab that option/value.

Comment (Jeff-Wang93) [inline vent/menus/main.py]: Ah I see. OK

**Pull request merged**